### ISSUE TITLE
Conda Installation Version Inconsistency

### ISSUE THREAD

**Issue by AndreasClenow** [quantopian/pyfolio#560]
Hi guys,

Seems to be an issue with the conda package version. Installing pyfolio through conda will only give you version 0.5.1, with no later version apparently available. 

Installing through pip on the other hand will give you version 0.9.0. 

It would be great if you could update the conda package, to keep things consistent.

Thanks,

Andreas

**Comment by twiecki** [comment]
Yes, unfortunately we haven't kept up-to-date on conda. Probably best to remove it for now.

**Comment by AndreasClenow** [comment]
Remove it from conda? That seems like a step in the wrong direction, Thomas. Updating the conda package would be a far more preferable solution.

**Comment by twiecki** [comment]
@richafrank How do we build new conda versions for empyrical, alphalens and pyfolio?
